test_conformance/compiler/test_compile.c

```diff
@@ -2519,6 +2519,33 @@ int test_execute_after_embedded_header_link(cl_device_id deviceID, cl_context co
 #include <direct.h>
 #endif
 
+/*
+ * Create a unique directory fooXXX in the current working directory. XXX is
+ * replaced with a three-letter string that makes the directory unique.
+ */
+static int mkdir_foo(char *foo)
```

Comment:
Could this function use 'mktemp' to generate unique filename?

Reply:
mktemp has race problem. mkdtemp could be used here, but it is only available in posix compliant systems.